jsettlers.graphics/src/jsettlers/graphics/localization/Labels.java

```diff
@@ -131,4 +131,9 @@ public static String getName(EActionType action) {
 	public static String getProgress(EProgressState loading) {
 		return getString("progress_" + loading);
 	}
+
+	public static String getName(EMaterialType material) {
+		// TODO Auto-generated method stub
```

Comment:
Wouldn't it be possible to just use https://github.com/jsettlers/settlers-remake/pull/236/files#diff-ab8ae3ce41667a0e8e2a3d65bb8974f8R105 ?